src/components/Doc/ComponentDoc.js

```diff
@@ -49,6 +50,22 @@ export const ComponentDoc = ({
         <title>{pageTitle || name}</title>
         <meta name="description" content={description} />
       </Helmet>
+      {stable && (
+        <Box pad={{ bottom: 'medium' }}>
+          <Box
+            alignContent="center"
+            direction="row"
+            background="light-3"
+            pad="xsmall"
```

Comment:
Text seems like it doesn't have enough room to breathe and could benefit from additional padding. Using Notification might help here.

Reply:
Switched to use Notification component